server/serializers.py

```diff
@@ -266,7 +266,13 @@ def jsonize_clientside_tab(tab: clientside.TabUpdate) -> Dict[str, Any]:
     return d
 
 
-def jsonize_i18n_message(message: I18nMessage, ctx: JsonizeContext) -> Optional[str]:
+def jsonize_i18n_message(message: I18nMessage, ctx: JsonizeContext) -> str:
+    """Localize (or unwrap, if it's a TODO_i18n) an `I18nMessage`
+    
+    Uses `locale_id` from `ctx`
+    
+    Raises `KeyError` if the message text cannot be found in the catalogs.
```

Comment:
In this particular case -- which is not general -- the `I18nMessage` comes from a _module_, not from Workbench. A bug in a module should not make Workbench crash.

How about: "If the message text is not in the user-locale catalog or the default-locale catalog, log a KeyError and return a JSON-serialized representation of the I18nMessage."

And: "If the message text in the default locale cannot be formatted correctly, log an ICUError and return a JSON-serialized representation of the I18nMessage."

Reply:
Well, a JSON-serialized representation would look really bad. On the other hand, it will contain the maximum amount of information possible for the user (at least if it does not get covered by other elements).

Reply:
Meh, I'm not opinionated about JSON-serializing. It's just as wrong as anything else. As long as we log the error, don't crash, and _do_ include the message ID, we'll be fine.